### ISSUE TITLE
Merging of ETL files has failed

### ISSUE THREAD

**Issue by etbuilder** [microsoft/ApplicationInsights-dotnet#1438]
If you are reporting bug/issue, please provide detailed Repro instructions.

## Repro Steps
1. Cannot reproduce locally

##  Actual Behavior
We are receiving an exception every couple of minutes:

Message: Merging of ETL files has failed (0x80070507) (Flags: 0x0000011f).
Exception type: System.Runtime.InteropServices.COMException	
MachineName "Removed"
AgentVersion:  2.4.60309.3
AgentSession: 8059820b-bb11-4778-9734-e3875840fcea |  
Description: StopProfiler failed.
RunningMode: Antares
Source: ApplicationInsightsProfiler

Exception:
```System.Runtime.InteropServices.COMException:
   at DiagnosticsHub.StandardCollector.Host.Interop.ICollectionSession.Stop (ApplicationInsightsProfiler, Version=2.4.60309.3, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at Microsoft.ServiceProfiler.Collectors.StandardCollectorBase.<StopAndCollectAsync>b__10_0 (ApplicationInsightsProfiler, Version=2.4.60309.3, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at System.Threading.Tasks.Task`1.InnerInvoke (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at System.Threading.Tasks.Task.Execute (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at Microsoft.ServiceProfiler.Collectors.DetailedTraceCollector+<StopCollectingAndSaveTrace>d__26.MoveNext (ApplicationInsightsProfiler, Version=2.4.60309.3, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at Microsoft.ServiceProfiler.Collectors.DetailedTraceCollector+<StopCollectingAndUploadTraceAsync>d__24.MoveNext (ApplicationInsightsProfiler, Version=2.4.60309.3, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at Microsoft.ServiceProfiler.Engine+<>c__DisplayClass1_1+<<RunAsync>b__2>d.MoveNext (ApplicationInsightsProfiler, Version=2.4.60309.3, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)
   at Microsoft.ServiceProfiler.Agent.Orchestration.Orchestrator+<TraceAsyncOperation>d__15.MoveNext (Microsoft.ServiceProfiler.Agent.Orchestration, Version=2.4.60309.3, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)```

## Extension Information
 Site Extension, version 2.5.4, is up to date.
 Agent version 2.4.60309.3 is installed and up to date.
 Application Insights Profiler Web Job is enabled. Disable
 Web Job monitoring is active. Stop monitoring


## Version Info 
SDK Version : .Net 4.7
.NET Version :	 						
How Application was onboarded with SDK: (Azure Extension) :
OS : Windows
Hosting Info (IIS/Azure WebApps/ etc) : Web app

**Comment by truptisatardekar** [comment]
Hello,

I'm facing similar issue. I don't have the site extension installed. 
![Untitled](https://user-images.githubusercontent.com/28647908/157903345-90da898d-541b-4fbd-80fd-56feaf103884.jpg)



Also when I try to add the extension, I entered "Application Insights extension for Azure App Service" in the search but I got no results.
Below is the configuration I have in the appservice -> app settings
![Screen Shot 2022-03-11 at 17 00 16](https://user-images.githubusercontent.com/28647908/157903153-44c651cf-a896-406c-a1c1-48422f773161.png)
